actors/common/src/main/java/org/sunbird/learner/util/SchedulerManager.java

```diff
@@ -18,6 +19,6 @@
   /** all scheduler job will be configure here. */
   public static void schedule() {
     service.scheduleWithFixedDelay(new DataCacheHandler(), 0, PAGE_DATA_TTL, TimeUnit.HOURS);
-    ProjectLogger.log("started scheduler job - org.sunbird.learner.util.SchedulerManager");
+    ProjectLogger.log("started scheduler job - for page cache reload.", LoggerEnum.INFO.name());
```

Comment:
From:
"started scheduler job - for page cache reload."

To:
"SchedulerManager:schedule: Started scheduler job for cache refresh."